Resources/NetHook2/SetupDependencies.cmd

```diff
@@ -1,7 +1,29 @@
 @echo off
-@powershell -NoProfile -ExecutionPolicy unrestricted -Command "%~dp0\InstallDependencies.ps1"
 
-pushd .
-cd %~dp0
-call BuildDependencies.cmd
-popd
\ No newline at end of file
+if not defined DevEnvDir (
+    for /f "usebackq tokens=*" %%i in (`"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe" -latest -products * -requires Microsoft.VisualStudio.Component.VC.Tools.x86.x64 -property installationPath`) do (
+        set InstallDir=%%i
+    )
+
+    if exist "%InstallDir%\Common7\Tools\vsdevcmd.bat" (
+        call "%InstallDir%\Common7\Tools\vsdevcmd.bat"
+    )
+)
+
+where.exe /q vcpkg
+if %ERRORLEVEL%==1 (
+    if not exist "vcpkg/" (
```

Comment:
I don't like this, can't we ask the user to have vcpkg installed?
Or is this for CI?aaa

Reply:
This is mostly for CI. For what it's worth, this copy of it only gets added into PATH for the duration of the session.

Reply:
I now realize that it does a full setup into the user's profile despite the binary living under the project folder. (This is my first time with vcpkg). That's indeed not ideal. I've updated it now to explicitly only do that for CI, else print out instructions for where to get it if missing.